tests/test_kernel32/test_process.py

```diff
@@ -240,3 +240,14 @@ def test_terminates_process(self):
         TerminateProcess(handle, 42)
         process.communicate()
         self.assertEqual(process.returncode, 42)
+
+
+class TestCreateToolhelp32Snapshot(TestCase):
+    """
+    Tests for :func:`pywincffi.kernel32.CreateToolhelp32Snapshot`
+    """
```

Comment:
Could you add one more test to make sure this raises an exception when CreateToolhelp32Snapshot is passed an invalid process id?  Any number not 0 or not divisible by 4 should work.